vhdbuilder/packer/install-dependencies.sh

```diff
@@ -77,12 +77,20 @@ else
   done
 fi
 
+mkdir -p /etc/systemd/system/chrony.service.d
+cat >> /etc/systemd/system/chrony.service.d/10-chrony-restarts.conf <<'EOF'
```

Comment:
I think we can also edit the first EOF to be `EOF` instead of `'EOF'`

Reply:
Done.